velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -29,8 +29,9 @@ bool validateColNames(const ::substrait::NamedStruct& schema) {
     for (auto i = 0; i < name.size(); i++) {
       auto c = name[i];
       if (!token.isUnquotedPathCharacter(c)) {
-        std::cout << "Illegal column charactor " << c << "in column " << name
-                  << std::endl;
+        std::cout
```

Comment:
Looks this should also be logged. Thanks!

Reply:
The `validateLog_` can't be accessed in this method so I think we just print it out will be okay.